test/http-fetcher-test.js

```diff
@@ -74,4 +74,27 @@ suite('http-fetcher-test', () => {
       });
     });
   });
+
+  test('it should allow setting custom headers at request time', () => {
+    const request = {
+      query: '{ shop { name } }',
+      variables: {}
+    };
+
```

Comment:
Please remove these new lines interspersed throughout the const declarations.

Reply:
Is this what you are looking for?
```javascript
test('it should allow setting custom headers at request time', () => {
    const request = {
      query: '{ shop { name } }',
      variables: {}
    };
    const customHeaders = {'X-API-KEY': '12345'};
    const fetcher = httpFetcher('https://graphql.example.com', {headers: customHeaders});
    const requestTimeHeaders = {Authorization: 'abcde'};

    return fetcher(request, requestTimeHeaders).then(() => {
      const [_url, {headers}] = fetchMock.lastCall();

      assert.deepEqual(headers, {
        'Content-Type': 'application/json',
        Accept: 'application/json',
        ...customHeaders,
        ...requestTimeHeaders
      });
    });
  });
```
Updated as such, it no longer matches the formatting of the other tests and the original spacing was a deliberate choice.